assemble/conf/accumulo.properties

```
@@ -31,3 +31,7 @@ instance.secret=DEFAULT
 
 ## Set to false if 'accumulo-util build-native' fails
 tserver.memory.maps.native.enabled=true
+
+## (optional) include additional property files for a resource group
+## based on the ACCUMULO_RESOURCE_GROUP env var set in accumulo-service
+includeOptional=group-${env:ACCUMULO_RESOURCE_GROUP}.properties
```

So there are no surprises, I'd leave the include line commented out.

```suggestion
#includeOptional=group-${env:ACCUMULO_RESOURCE_GROUP}.properties
```

Also, did you get a chance to test this to verify it worked?

---

I didn't test it. I feel like that's testing commons-configuration's documented behavior.

---

Actually, could provide both examples, so users can decide whether the include is optional or not.

```suggestion
#include=group-${env:ACCUMULO_RESOURCE_GROUP}.properties
#includeOptional=group-${env:ACCUMULO_RESOURCE_GROUP}.properties
```

---

Suggestion applied in 0bc61ef